src/ensembl/src/shared/components/question-button/QuestionButton.tsx

```diff
@@ -20,9 +29,19 @@ const QuestionButton = (props: Props) => {
     setIsHovering(false);
   };
 
+  const styles = props.classNames
+    ? { ...defaultStyles, ...props.classNames }
+    : defaultStyles;
+
+  const classParams = props.styleOption
+    ? [defaultStyles.default, styles[props.styleOption]]
+    : defaultStyles.default;
+
+  const className = classNames(classParams);
```

Comment:
The `classNames` library can take an arbitrary number of arguments, which can be either simple strings, or hashes, with keys being class names and values being whether to add them or not.

This will compose in a nicer statement like so:

```
 const className = classNames(
  defaultStyles.default,
  {
    [defaultStyles[props.styleOption]]: props.styleOption
  },
  props.className
);
```